jsonnet/telemeter/client/kubernetes.libsonnet

```diff
@@ -208,7 +208,7 @@ local securePort = 8443;
               port: 'https',
               scheme: 'https',
               tlsConfig: {
-                caFile: '/etc/prometheus/configmaps/telemeter-client-serving-certs-ca-bundle/%s' % servingCertsCABundleFileName,
```

Comment:
hmm but we have a configmap with that name: https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/telemeter-client/serving-certs-c-a-bundle.yaml#L8. Maybe the issue is that we simply need to tadd that configmap to the prometheus-k8s manifest so it is loaded in the statefusets

Reply:
but it's the same ca for all serving certs, why multiple mounts then?

Reply:
yes this is true these bundle's will be the same. I think there is some justification to having each component be able to ship all of the dependencies needed to use/interact with it: "I am prometheus/I am telemeter and these are the CA's you need if you want to interact with me". Maybe the relationship with the CA bundle should be the inverse, anything that needs to interact with a service needs to provision its own bundle, in which case we should have a "cluster-monitoring-serving-certs-ca-bundle".

Reply:
Yeah I can see both sides as well, my biggest argument (currently) for having a single cert (probably even more generic along `serving-certs-ca-bundle`, is what I'm thinking), is because the control of mounting additional CAs is on the Prometheus side. I'm happy to rename the mount, I would start in this PR and adapt everywhere in the PR that would integrate this into the cluster-monitoring-operator. Sounds good?

Reply:
sounds great. let's simplify